### DISCUSSION TITLE
Cant login to Gitbook on Chrome

### DISCUSSION THREAD

**Discussion by GG-DAO** [community/community#29211] [Other Feature Feedback, Questions, & Ideas]
Ive been trying to login to Gitbook on Chrome for months now to no avail. Whenever I try to login I receive the same error that you can see in the linked image

![Capture](https://user-images.githubusercontent.com/98059703/184259817-51e0f24a-2605-45d0-b411-31ccc652a761.PNG)

Ive tried resetting cookies and other basic troubleshooting. Nothing seems to work. I am able to login fine from Opera or other browsers, but not Chrome. I also have to hard refresh when viewing other Gitbooks whenever I try to switch between pages. Its extremely inconvenient & frustrating.

**Comment by surachittham** [comment]
This is open since Aug 2022? And you have been trying since months? I just started having this issue, not sure if there is a solution. Will deleting all history and cookies help? I wouldn't be comfortable doing it though.

**Comment by verdantjuly** [comment]
![스크린샷 2023-08-08 오전 7 57 44](https://github.com/community/community/assets/131671804/db76d57d-073e-4c0c-904e-f77feb1d6b0b)

same here in August, 2023.
I checked online connection and have done basic troubleshootings including reomving all cookies. But I can't access login page through Chrome.

I'm using Safari to access my document now.

**Reply by verdantjuly** [reply]
Solved issue by reinstalling Chrome!

**Comment by glewe** [comment]
Same here in both, Chrome and Firefox. Edge works though.
Is there any more info available about how to fix this?

**Comment by Stephcraft** [comment]
Same problem on Brave browser.

**Comment by 3h4x** [comment]
wtf. it still doesn't work on brave in 2024.
it works in safari...
